config/simorgh/simorgh.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+module.exports = {
+  baseUrl: 'http://localhost:7080',
+  paths: [
+    '/news/articles/c0000000025o',
+    '/news/articles/c0000000027o'
+  ],
+  LIGHTHOUSE_OPTS: {
+    config: {
+      extends: 'lighthouse:default',
```

Comment:
Do we really need to expose the full Lighthouse options structure? I think I'd prefer to extract all that way so that people need very little to no knowledge of how Lighthouse works. I'd suggest we just have the categories option so it looks like this:

```js
module.exports = {
  baseUrl: 'http://localhost:7080',
  paths: [
    '/news/articles/c0000000025o',
    '/news/articles/c0000000027o'
  ],
  lighthouseCategories = ['accessibility', 'best-practices', 'seo', 'pwa']
}
```

(where lighthouseCategories defaults to `['accessibility']` in the main code)